src/scenes/UserDetail/components/UserProfile.js

```diff
@@ -216,152 +293,203 @@ export default function UserProfile(props) {
                 spacing={3}
             >
                 {header}
-                {editToggle}
+                {editNameToggle}
             </Stack>
             <Divider />
+            {oldState.contact && (
+                <Stack
+                    direction={"row"}
+                    justifyContent={"space-between"}
+                    alignItems={"top"}
+                    spacing={3}
+                >
+                    <Typography>Contact Information</Typography>
+
+                    {editContactToggle}
+                </Stack>
+            )}
             <Box sx={{ width: "100%" }}>
-                {Object.keys(fields).map((key) => {
-                    if (editMode) {
-                        return (
-                            <TextField
-                                value={state[key]}
-                                variant={"standard"}
-                                fullWidth
-                                label={fields[key]}
-                                id={key}
-                                onChange={(e) => {
-                                    toChange.current = {
-                                        ...toChange.current,
-                                        [key]: e.target.value,
-                                    };
-                                    setState({
-                                        ...state,
-                                        [key]: e.target.value,
-                                    });
-                                }}
-                            />
-                        );
-                    } else {
+                <Stack direction={"row"} justifyContent={"space-between"}>
+                    <Typography>{fields.name}</Typography>
+                    <Typography>{oldState.name}</Typography>
```

Comment:
@duckbytes On the user I was logged in as, it didn't have any contact information, thus the edit button won't show but their name is still rendered. Is this the expected logic?
<img width="1269" alt="Screen Shot 2022-05-17 at 4 12 32 AM" src="https://user-images.githubusercontent.com/51278228/168890592-1e943d77-161b-4351-8050-6f5f4b246891.png">